.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
```

Comment:
I noticed that for techLag we have a pre-push commit of `npm test` and a pre-commit of `npm run lint`.

I definitely think we should have `npm run lint` in one of those two places. Maybe add it to the pre-commit for now and @kevindigo do you have a preference for whether these files look like they do in techlag or if we keep both commands in pre-commit?

Reply:
I'm also noticing that package.json has 
    "husky": {
        "hooks": {
            "pre-commit": "npm run lint"
        }
        on lines10-12.
        
        Now that we're using Husky 5, I think maybe you get rid of this in package.json and instead add linting to the pre-commit file

Reply:
found this in the docs that will affect our work:

"if your package is not private and you're publishing it on a registry like npmjs.com, you need to disable postinstall script using pinst. Otherwise, postinstall will run when someone installs your package and result in an error."

script would look like:
// package.json
{
  "private": false,
  "scripts": {
    "postinstall": "husky install",
    "prepublishOnly": "pinst --disable",
    "postpublish": "pinst --enable"
  }
}

Reply:
1. I think we should do lint and test pre-commit. 
2. I think the husky section of package.json should go away with v5. 
3. Good catch on the postinstall issue. The script fragment doesn't seem to be doing what the description above it says is necessary, but it might be correct.